src/optional.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_OPTIONAL_H
+#define BITCOIN_OPTIONAL_H
+
+#include <boost/none.hpp>
+#include <boost/optional/optional.hpp>
```

Comment:
Maybe replace these two lines as `#include <boost/optional.hpp>` to pass travis tests.

In commit 'Remove uses of chainActive and mapBlockIndex in wallet code '